src/RandomForest.jl

```diff
@@ -69,9 +84,23 @@ function predict(forest::RandomForest, data::AbstractMatrix)
     end
 end
 
+"""
+    $(SIGNATURES)
+
+This function recursively prints the structure of the ForestTree. It's primarily used for debugging and visualizing the Forest structure.
+
+## Arguments
+- `io::IO`: The IO context to print the Forest structure.
+- `forest::RandomForest`: The RandomForest to be printed.
+
+## Returns
+- `Nothing`: This function prints the structure of the `RandomForest`.
+"""
 function Base.show(io::IO, forest::RandomForest)
     for (index, tree) in enumerate(forest.trees)
         println(io, "Tree $index")
         print(tree)
+        println(io, "Tree $index")
```

Comment:
This also got duplicated

Reply:
Okay, thank you, I will look into it again :) Merging with the docstrings was a little painful